code/game/turfs/simulated/outdoors/sidewalk.dm

```diff
@@ -0,0 +1,33 @@
+/decl/flooring/outdoors/sidewalk
+	name = "sidewalk"
+	desc = "Concrete shaped into a path!"
+	icon = 'icons/turf/outdoors.dmi'
+	icon_base = "sidewalk"
+	footstep_sounds = list("human" = list(
+		'sound/effects/footstep/LightStone1.ogg',
+		'sound/effects/footstep/LightStone2.ogg',
+		'sound/effects/footstep/LightStone3.ogg',
+		'sound/effects/footstep/LightStone4.ogg',))
+
+/turf/simulated/floor/outdoors/sidewalk/Initialize(mapload)
+	var/possibledirts = list(
```

Comment:
You missed one!

Reply:
turns out the entire contents of sidewalk.dm was duplicated in dirt.dm for no reason so I didn't change that version.